files/build_templates/docker_image_ctl.j2

```diff
@@ -34,14 +34,14 @@ function updateHostName()
 function getBootType()
 {
     local BOOT_TYPE
-    case "$(cat /proc/cmdline | grep -o 'SONIC_BOOT_TYPE=\S*' | cut -d'=' -f2)" in
-    warm*)
+    case "$(cat /proc/cmdline)" in
+    *warm*)
         TYPE='warm'
         ;;
-    fastfast)
+    *fastfast*)
         TYPE='fastfast'
         ;;
-    fast*)
+    *fast*)
```

Comment:
>*fast* [](start = 4, length = 6)

This one is also too general. We can match for 2 patterns:
1. `*SONIC_BOOT_TYPE=fast*`
2. `*fast-reboot*`